packages/components/src/tabs/tablist.tsx

```diff
@@ -42,8 +44,7 @@ export const TabList = forwardRef<
 		return null;
 	}
 	const { store } = context;
-
-	const { activeId, selectOnMove } = store.useState();
+	const { activeId, selectOnMove } = tabStore as Ariakit.TabStoreState;
 	const { setActiveId } = store;
```

Comment:
This can be changed to 
```suggestion
	const { setActiveId } = context.store;
```

and then we can delete `const { store } = context;` two lines above

Reply:
We could, but I preferred to leave it, because we need `store` separately as we're passing it as a separate prop to `Ariakit.TabList`.